main/src/main/scala/org/clulab/numeric/mentions/MeasurementMention.scala

```diff
@@ -15,7 +15,7 @@ class MeasurementMention ( labels: Seq[String],
                            val value: Option[Seq[String]],
                            val unit: Option[Seq[String]],
                            val fromRange: Boolean,
-                           unitNormalizer: UnitNormalizer)
+                           unitNormalizer: BaseUnitNormalizer = NullUnitNormalizer)
```

Comment:
@maxaalexeeva, there is a default value so that habitus is at least syntactically compatible.